gradle.properties

```diff
@@ -0,0 +1,16 @@
+GROUP=com.pinterest.ktlint
+VERSION_NAME=0.3.4-SNAPSHOT
+
+POM_DESCRIPTION=An anti-bikeshedding Kotlin linter with built-in formatter.
+
+POM_URL=https://github.com/pinterest/ktlint
+POM_SCM_URL=https://github.com/pinterest/ktlint
+POM_SCM_CONNECTION=scm:git:git://github.com/pinterest/ktlint.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/pinterest/ktlint.git
+
+POM_LICENCE_NAME=MIT
```

Comment:
typo: `s/LICENCE/LICENSE`

Reply:
This is from - https://github.com/square/okhttp/blob/master/gradle.properties#L11.

Also from the plugin you mentioned: https://github.com/vanniktech/gradle-maven-publish-plugin/commit/a8d849159f6638549f5e0bdbc1b51c4455538b92#diff-a58d7d71ad884cc5fd549abef2a675eeR40.

Reply:
Yes, I saw, but these are just internally-used properties, correct? There doesn't seem to be a good reason to use the British-English spelling of 'license', especially if we're going to use a custom-defined task instead of a plugin. If we switched to use a plugin, I'd understand it's outside of our control.

Reply:
I think going your way, using the plugin is better since it handles dokka and encapsulates the functionality in the script I was using before.

Reply:
👍